mongo/integration/collection_test.go

```diff
@@ -74,6 +74,50 @@ func TestCollection(t *testing.T) {
 			assert.True(mt, ok, "expected error type %v, got %v", mongo.WriteException{}, err)
 			assert.NotNil(mt, we.WriteConcernError, "expected write concern error, got %+v", we)
 		})
+
+		mt.Run("options are converted", func(mt *mtest.T) {
```

Comment:
nit: Skip this top level test for server versions < 3.2 rather than skipping all sub-tests individually.

Reply:
Done.